### ISSUE TITLE
Line drawing macOS has strange gaps and looks different from linux

### ISSUE THREAD

**Issue by abdul-malek-octane** [tarkah/tickrs#142]
Every 4 dots (vertically) there is a dot missing. Line looks patchy and thinner than on macOS for the same font, same vertical horizontal spacing. 
![Screen Shot 2023-03-21 at 10 54 33 AM](https://user-images.githubusercontent.com/95316870/226648409-f6be1968-5b5a-44d2-8bb9-e2511a747390.png)

**Comment by tarkah** [comment]
This is a font issue. Use a terminal where you can configure line height or y-offset and lower it. I do this with Alacrity to remove gaps with my preferred font.

**Comment by abdul-malek-octane** [comment]
I also use Alacritty. y-offset does not fix this no matter what unfortunately. I've played around with it quite a bit. Also worth noting that the same y-offset settings on Alacritty render differently on macOS vs linux (ubuntu, gnome). 

The above picture is the best I could get while playing with y-offset. Without y-offset, the difference is much more apparent:
![Screen Shot 2023-03-21 at 11 06 59 AM](https://user-images.githubusercontent.com/95316870/226650176-4b69585f-cf3a-4894-a352-5a8694b9f4ae.png)

**Comment by abdul-malek-octane** [comment]
Also even with no offsets, the gap issue is not present on Linux, so I believe it is an OS-related rending issue, at least in part.

**Comment by tarkah** [comment]
Are you using identical fonts between Linux and Macos? 

If you're using alacrity on both platforms, it is most likely an upstream issue there.

**Comment by abdul-malek-octane** [comment]
Yep, identical fonts across both Linux and macOS. Hm, I don't think it's an Alacritty either since the issue occurs on other terminal emulators as well on macOs but not on Linux.

For example:
- macOS
  - Alacritty (issue occurs) (with and without y-offset)
  - Terminal (issue occurs) (no y-offset)
  - iTerm (issue occurs) (no y-offset)
- Linux
  - Alacritty (no issue) (with and without y-offset)
  - Gnome Terminal (no issue) (no y-offset)
  - Kitty (no issue) (no y-offset)
